src/components/DatatableHeader/index.tsx

```diff
@@ -10,14 +10,14 @@ const DataTableHeader = ({ resource, downloadURL, jsonUrl } : {resource: Resourc
   return (
     <div className="ds-l-row ds-u-align-items--center ds-u-margin-bottom--2">
       <div className="ds-l-col--12 ds-u-display--flex ds-u-flex-wrap--wrap ds-u-justify-content--between ds-u-align-items--center">
-        <div className="ds-l-col--12 ds-l-lg-col--6">
+        <div className="ds-l-col--12 ds-l-lg-col--4">
```

Comment:
Were these col width changes (here and below) intentional to accommodate the larger button text, or a regression? This width was just changed last week to address another issue.